test/hotspot/gtest/classfile/test_symbolTable.cpp

```diff
@@ -139,3 +144,27 @@ TEST_VM(SymbolTable, test_cleanup_leak) {
 
   ASSERT_EQ(entry2->refcount(), 1) << "Symbol refcount just created is 1";
 }
+
+TEST_VM(SymbolTable, test_cleanup_delay) {
```

Comment:
Please another test that checks that `drain_cleanup_delay_queue` also does decrement refcounts?

Reply:
Good idea, will add